azavea/raster-foundry
# Remove unique constraint and update importers

## Overview

Remove unique scene_name_org_datasource constraint on scenes
Update importers to check if the user has created a scene with the same name already before inserting

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

## Testing Instructions
* Run migrations
 * Verify that the new/old landsat 8 importers, as well as the sentinel 2 importers will not try to import the same scene twice if run for the same day multiple times
* Verify that you can create scenes with the same name manually (not through the importer)

Closes #1610
